db/migrate/20160805000209_create_contacts.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContacts < ActiveRecord::Migration
+  def change
+    create_table :contacts do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :phone
+      t.string :service
+      t.date :date
+      t.string :availability
+      t.integer :skillset
+      t.text :notes
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
We can do this.  Let me know what you think should be required.  I think
first name or last name (may not have both), service, and I am not sure
what else you are sure that you have every time.


On Fri, Aug 5, 2016 at 8:15 PM, Rylan Bowers <notifications@github.com>
wrote:

> In db/migrate/20160805000209_create_contacts.rb
> <https://github.com/LaurieLinz/organize-your-network/pull/1#discussion_r73779112>
> :
>
> > +  def change
> > +    create_table :contacts do |t|
> > +      t.string :first_name
> > +      t.string :last_name
> > +      t.string :email
> > +      t.string :phone
> > +      t.string :service
> > +      t.date :date
> > +      t.string :availability
> > +      t.integer :skillset
> > +      t.text :notes
> > +
> > +      t.timestamps null: false
> > +    end
> > +  end
> > +end
>
> I think its a good practice to add null: false for things you are going to
> require. It provides a failing level of check at the database level and is
> good design, in my opinion. Reinforces the validates_presence_of when you
> add 'em in. We didn't really discuss what is or isn't required, though. No
> big deal.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/LaurieLinz/organize-your-network/pull/1/files/9ef65a11f5f83a5f7a0d94b04cf928ba6d4ef348#r73779112>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOKlgyf18HCTcVNsULZwDhLgMzoOBLNlks5qc-5ZgaJpZM4JdP6T>
> .
>



-- 

303.601.8014

www.LaurieLinz.com <http://www.laurielinz.com/>

Github:https://github.com/LaurieLinz

Blog:https://laurielinz.wordpress.com/

LinkedIn: https://linkedin.com/in/lauralinz

Mail: lalinz303@gmail.com